enclave_wrangler/dataset_download.py

```diff
@@ -70,8 +71,9 @@ def views2(datasetRid: str, endRef: str) -> [str]:
     file_parts = [f['logicalPath'] for f in response_json['values']]
     return file_parts[1:]
 
+
 @typechecked
-def datasets_views(datasetRid: str, file_parts: [str]) -> None:
+def datasets_views(datasetRid: str, file_parts: [str]) -> pd.DataFrame:
```

Comment:
In order to re-use this function, and since I didn't need it to do `df.to_csv()`, I changed it to return a `df`, and moved your `df.to_csv()` in the calling function.